generators/generator-bot-enterprise-assistant/README.md

```diff
@@ -1,68 +1,50 @@
-- See examples of more complex conversational flows, and more advanced language understanding and generation
-- See an example of a root bot and skills handling complex user interactions including interruptions and Adaptive Cards
-
-## Packages
-
-Your bot can use the [Azure Bot Framework component model](https://aka.ms/ComponentTemplateDocumentation) to extend the base functionality. From Composer, use the Package Manager to discover additional packages you can add to your bot.
-
-This bot starts with the following packages:
-
-- [Help and Cancel intent handling](https://www.nuget.org/packages/Microsoft.Bot.Components.HelpAndCancel/)
-- [Orchestrator](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Orchestrator/)
-
-## Supported Languages
+## Included packages
 
-- English (en-US)
+- [Help and Cancel Intent Handling](https://www.nuget.org/packages/Microsoft.Bot.Components.HelpAndCancel/)
+- [Bot Framework Orchestrator](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Orchestrator/)
 
-## Azure Resource Deployment
+The Enterprise Assistant Bot uses packages to extend its capabilities. [Learn more](https://aka.ms/ComponentTemplateDocumentation).
 
-To run this bot you'll need the resources listed below. Create a publishing profile in Composer to provision and publish to your Azure resources for your bot.
+### Required Azure resources
 
-- Azure Bot Registration configured with Microsoft Azure Active Directory authentication with access to the following scopes:
+- Azure Bot Service Registration configured with Microsoft Azure Active Directory authentication with access to the following scopes:
     - Calendars.ReadWrite
     - Contacts.Read
     - Directory.Read.All
     - People.Read
     - People.Read.All
     - User.ReadBasic.All
     - User.Read.All
-- [Language Understanding (LUIS)][luis], or another recognizer of your choice
-- [QnA Maker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/overview/overview)
+- [Azure Language Understanding (LUIS)][luis], or another recognizer of your choice
+- [Azure QnA Maker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/overview/overview)
 - A storage solution for persistent state storage like Azure Cosmos DB
 
-## Using this template
-
-### From Composer
-
-From Composer you'll use the **New** button on the **Home** screen to create a new bot. After creation, Composer will guide you through making customizations to your bot. If you'd like to extend your bot with code, you can open up your bot using your favorite IDE (like Visual Studio) from the location you choose during the creation flow.
-
-### From the command-line
+## Supported Languages
 
-This template can also be installed from the [command line](https://github.com/microsoft/botframework-components/blob/main/generators/command-line-instructions).
+- English (en-US)
 
 ## License
```

Comment:
I believe the heading size for this particular readme is not consistent with the calendar and people, and perhaps other readmes as well which has the 3 "#"s